playground/infrastructure/ci_cd.py

```diff
@@ -84,9 +84,12 @@ def _run_ci_cd(step: config.Config.CI_CD_LITERAL, sdk: Sdk):
   logging.info("Number of found Playground examples: %s", len(examples))
 
   if step == config.Config.CI_STEP_NAME:
-    logging.info("Start of verification Playground examples ...")
-    _ci_step(examples=examples)
-    logging.info("Finish of verification Playground examples")
+    single_file_examples = [example for example in examples
+                            if example.tag.multifile is False]
```

Comment:
Let's move this part to the `ci_helper.verify_examples` function.

Reply:
Done.